src/reward.ts

```diff
@@ -69,6 +69,11 @@ export default class Reward {
     return rewards.map(reward => new Reward(-reward.count, reward.type));
   }
 
+  static discount(rewards: Reward[], multiplier: number, type: Resource): Reward[] {
```

Comment:
Should add a comment, like:

`/** For each reward, if it is of corresponding type, reduce the cost by dividing it by multiplier */`

I would also switch the order of the type / multiplier arguments.

Reply:
@coyotte508 : are you using ```tokenModifier``` in the expansion? If not, I would remove it as it's used only for Nevlas with PI

Reply:
Nope, not using it. For now it could be replaced with faction === Nevlas ? 2 : 1.

Later we can make it more flexible if need be.

Reply:
I need this PR for the range extension when moving spaceships in the expansion (it uses power)